opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -297,6 +297,7 @@ namespace Opm
           for the schedule instances created by loading a restart file.
         */
         static bool cmp(const Schedule& sched1, const Schedule& sched2, std::size_t report_step);
+        void applyKeywords(const Deck &deck, std::size_t timeStep);
```

Comment:
I tried to use the Pybind11 [deck_keyword.cpp](https://github.com/OPM/opm-common/blob/master/python/cxx/deck_keyword.cpp) file like this

```python
    wconprod = DeckKeyword(
        parser["WCONPROD"],
        [[well_name, well_status, control_mode, oil_target, "*",
          "*", "*", "*", bhp_limit]],
        active_unit_system, default_unit_system)
    schedule.insert_keywords([wconprod], step=report_step)
```
see updated commit for this PR. The problem was that `deck_keyword.cpp` did not support `UDAValue` arguments (which is used by `WCONPROD`). I tried to fix it in an ad-hoc fasion, see last commit, but currently it
does not work. For example:

```python
from opm.io.parser import Parser
from opm.io.ecl_state import EclipseState
from opm.io.schedule import Schedule
from opm.io.summary import SummaryConfig
from opm2.simulators import BlackOilSimulator

deck  = Parser().parse('SPE1CASE1.DATA')
state = EclipseState(deck)
schedule = Schedule( deck, state )
report_step = 3
oil_target = 30000  # stb/day
well_name = "PROD"
well_status = "OPEN"
control_mode = "ORAT"
bhp_limit = 1000  # psia
wconprod = DeckKeyword(
     parser["WCONPROD"],
     [[well_name, well_status, control_mode, oil_target, "*",
          "*", "*", "*", bhp_limit]],
     active_unit_system, default_unit_system)
schedule.insert_keywords([wconprod], step=report_step)
summary_config = SummaryConfig(deck, state, schedule)
sim = BlackOilSimulator( deck, state, schedule, summary_config  )
sim.step_init()
sim.step()
sim.step()
sim.step()
sim.step()
sim.step()
sim.step()
sim.step_cleanup()
```

gives a zero oil production rate after report step 3, instead of the expected rate of 30000 stb/day.